### PR TITLE
Transit

### PR DESCRIPTION
**Pull request by lisamartelly**
:)

### CODE REVIEW

File: src/BusStop.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        if (json.departures[i].actual) {
+          tempActualDepartures.push(json.departures[i])
+        } else {
+          tempScheduledDepartures.push(json.departures[i])
+        }
+      }
+    //   console.log("ACTUAL Departures: ");
+    //   console.log(tempActualDepartures);
+    //   console.log("SCHEDULED TRANSIT: ");
+    //   console.log(tempScheduledDepartures);
```
Comment (danmartelly) [inline src/BusStop.js:25]: delete the zombie code

Comment (danmartelly) [inline src/BusStop.js:25]:
```suggestion

```
Delete the zombie code

File: src/BusStop.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import { useState, useEffect } from 'react';
+
+function BusStop(props) {
+    const [actualDepartures, setActualDepartures] = useState([]);
```
Comment (danmartelly) [inline src/BusStop.js:4]: I think this variable name is ok, but just wanted to suggest using "liveDepartures" instead

File: src/Transit.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import BusStop from './BusStop.js'
+
+
+function Transit() {
+  
+    return(<div>
+      <h1>TRANSIT</h1>
+      <BusStop stopName="Hennepin & 24th" stopId="1328" />
```
Comment (danmartelly) [inline src/Transit.js:8]: UI question: will the 2 bus stops be distinguishable to the user?

File: src/Weather.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    const [queryTime, setQueryTime] = useState(null);
+    const url="https://api.weather.gov/gridpoints/MPX/106,70/forecast";
+  
+    const fetchWeather = async () => {
+      const response = await fetch(url);
+      const json = await response.json();
+      const tempWeather = json.properties.periods.slice(0,2);
+      const tempUpdateTime = new Date(json.properties.updateTime).toLocaleTimeString();
+      const tempQueryTime = new Date(json.properties.generatedAt).toLocaleTimeString();
+      console.log(json.properties);
```
Comment (danmartelly) [inline src/Weather.js:16]: Should delete console log

File: src/Weather.js [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  
+  
+      // create celsius temperature from fahrenheit
```
Comment (danmartelly) [inline src/Weather.js:19]: :smile:

File: src/Weather.js [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+      setUpdateTime(tempUpdateTime);
+      setQueryTime(tempQueryTime);
+    };  
+  
+    useEffect(() => {
+      fetchWeather();
+    }, []);
+  
+    console.log("WEATHER");
+    console.log(weather);
```
Comment (danmartelly) [inline src/Weather.js:33]: delete

File: src/Weather.js [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+    return(<div>
+      <h1>WEATHER:</h1>
+      {weather.map(function(weatherObj) {
+              return (
+                <div>
+                  <h2>{weatherObj.name}</h2>
+                  <h3>Temperature: {weatherObj.temperature} {weatherObj.temperatureUnit} / {weatherObj.celsius} C</h3>
```
Comment (danmartelly) [inline src/Weather.js:40]: Suggestion: Show F for Fahrenheit?

File: src/Weather.js [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+                  <p>{weatherObj.detailedForecast}</p>
+                </div>
+                )
+              })}
+        <div>
+          <p>last queried: {queryTime}</p>
+          <p>last updated: {updateTime}</p>
```
Comment (danmartelly) [inline src/Weather.js:47]: nice touch on having the last updated. By the way, what's the difference between updated and queried?

**Review by danmartelly — APPROVED** [review]
A nit or two and some questions, but happy to approve already

File: src/BusStop.js [hunk 9]
```diff
@@ -0,0 +1,54 @@
+import { useState, useEffect } from 'react';
+
+function BusStop(props) {
```
Comment (danmartelly) [inline src/BusStop.js:3]: Maybe destructure props? At least that's my style preference

File: src/BusStop.js (hunk 1 above)
Comment (lisamartelly) [reply src/BusStop.js:25]: deleted

File: src/BusStop.js (hunk 2 above)
Comment (lisamartelly) [reply src/BusStop.js:4]: changed to "realTimeDepartures"

File: src/Transit.js (hunk 3 above)
Comment (lisamartelly) [reply src/Transit.js:8]: updated names to be more clear

**Pull request merged**